code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -44,3 +44,222 @@
+
+/mob/living/simple_animal/hostile/retaliate/clown/destroyer
+	name = "The Destroyer"
+	desc = "An ancient being born of arcane honking."
+	icon = 'icons/mob/Clown_mobs.dmi'
+	icon_state = "destroyer"
+	icon_living = "destroyer"
+	mob_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
+	turns_per_move = 5
+	response_help = "tries desperately to appease"
+	response_disarm = "bounces off of"
+	response_harm = "bounces off of"
+	speak = list("HONK!!!", "The Honkmother is merciful, so I must act out her wrath.", "parce mihi ad beatus honkmother placet mihi ut peccata committere,", "DIE!!!")
+	emote_see = list("honks", "sweats", "grunts")
+	speak_chance = 10
+	a_intent = INTENT_HARM
+	maxHealth = 800
+	health = 800
+	pixel_x = -16
+	speed = 6
+	harm_intent_damage = 50
+	melee_damage_lower = 40
+	melee_damage_upper = 60
+	attacktext = "Acts out divine vengeance on"
+	attack_sound = 'sound/items/bikehorn.ogg'
+	obj_damage = 50
+	environment_smash = ENVIRONMENT_SMASH_RWALLS
+	del_on_death = 5
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/human, /obj/effect/particle_effect/foam, /obj/item/soap)
+
+/mob/living/simple_animal/hostile/retaliate/clown/mutant
+	name = "Unknown"
+	desc = "Kill it for its own sake."
+	icon = 'icons/mob/Clown_mobs.dmi'
+	icon_state = "mutant"
+	icon_living = "mutant"
+	mob_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
+	turns_per_move = 5
+	response_help = "reluctantly sinks a finger into"
+	response_disarm = "squishes into"
+	response_harm = "squishes into"
+	speak = list("aaaaaahhhhuuhhhuhhhaaaaa", "AAAaaauuuaaAAAaauuhhh", "huuuuuh... hhhhuuuooooonnnnkk", "HuaUAAAnKKKK")
+	emote_see = list("squirms", "writhes", "pulsates", "froths", "oozes")
+	speak_chance = 5
+	a_intent = INTENT_HARM
+	maxHealth = 130
+	health = 75
+	pixel_x = -16
+	speed = 2
+	harm_intent_damage = 10
+	melee_damage_lower = 10
+	melee_damage_upper = 30
+	attacktext = "awkwardly flails at"
+	attack_sound = 'sound/items/bikehorn.ogg'
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/xeno/bodypartless, /obj/effect/particle_effect/foam, /obj/item/soap, /obj/effect/gibspawner/generic, /obj/effect/gibspawner/generic/animal, /obj/effect/gibspawner/human/bodypartless, /obj/effect/gibspawner/human)
+
+/mob/living/simple_animal/hostile/retaliate/clown/blob
+	name = "Something that was once a clown"
+	desc = "A grotesque bulging figure far mutated from it's original state."
+	icon = 'icons/mob/Clown_mobs.dmi'
+	icon_state = "Blob"
+	icon_living = "Blob"
+	mob_biotypes = list(MOB_ORGANIC, MOB_HUMANOID)
+	turns_per_move = 10
+	response_help = "reluctantly sinks a finger into"
+	response_disarm = "squishes into"
+	response_harm = "squishes into"
+	speak = list("hey, buddy", "HONK!!!", "H-h-h-H-HOOOOONK!!!!", "HONKHONKHONK!!!", "HEY, BUCKO, GET BACK HERE!!!", "HOOOOOOOONK!!!")
+	emote_see = list("jiggles", "wobbles")
+	speak_chance = 20
+	a_intent = INTENT_HARM
+	maxHealth = 130
+	health = 200

```

Comment:
making health higher than maxhealth is probably a bad idea

Reply:
whoops, thanks for pointing that out